core-web/src/main/java/org/silverpeas/core/web/treemenu/process/TreeHandler.java

```diff
@@ -67,6 +71,17 @@ public static String ProcessMenu(HttpServletRequest request, String menuType) {
     return TreeMenuJSON.getListAsJSONArray(items.getChildren());
   }
 
+  /**
+   * get information from request and build a level menu
+   * @param request
+   * @param menuType
+   * @return
+   * @throws RemoteException
+   */
```

Comment:
The parameters and the return value miss description.
The RemoteException isn't thrown by the method and hence it can be removed

Reply:
Done.